tests/integration/plots/test_plots.py

```diff
@@ -503,6 +503,7 @@ def test_repo_with_dvclive_plots(tmp_dir, capsys, repo_with_dvclive_plots):
 
 @pytest.mark.vscode
 def test_nested_x_defn_collection(tmp_dir, dvc, scm, capsys):
+    # https://github.com/iterative/dvc/pull/10318
```

Comment:
```suggestion
```

Reply:
that's not exactly relevant, but important to have to my mind - the test is advanced and complicated

Reply:
In that case, the test should be improved. Or, we can replace the link with comments/docstrign on what we are testing.

Reply:
sure, if you feel strong / consider this a blocker I'm 100% you removing it 🙏 

(why I do add this - because it was saving me tons and tons of time in different codebases and was very thankful for people who did put a link to an issue or PR. docstring is not the same - less context and duplicate pretty much what is already inside issue / PR anyways).

again, @amritghimire - 100% your call here, I don't have a strong opinion on this.

Reply:
Docstrings/comments can go stale, but you can at least update it. PR descriptions can get stale very fast.

Reply:
Removed.